### PR TITLE
Pull Request of Cloud buffer into Main

### PR DESCRIPTION
**Pull request by KirchnerMax**

### CODE REVIEW

File: src/cloudBuffer/buffer.py [hunk 1]
```diff
@@ -21,6 +32,15 @@ def append(self, measurement: str,
                tags: Union[List[Tuple[str, str]], None],
                value: Any,
                timestamp: Any):
+        """"
+        Function which adds a measurement point at the end of the private list
+        buffer.
```
Comment (OParczyk) [inline src/cloudBuffer/buffer.py:37]:
```suggestion
        Adds a measurement point to the end of the buffer.
```

File: src/cloudBuffer/buffer.py [hunk 2]
```diff
@@ -46,6 +66,12 @@ def append(self, measurement: str,
 
     def append_many(self, raw_point_list: List[
             Tuple[str, Union[List[Tuple[str, str]], None], Any, Any]]):
+        """"
+        Function which adds multiple measurement points at the end of the
+        private list buffer.
```
Comment (OParczyk) [inline src/cloudBuffer/buffer.py:71]:
```suggestion
        Adds multiple measurement points to the end of the buffer.
```

File: src/cloudBuffer/buffer.py [hunk 3]
```diff
@@ -46,6 +66,12 @@ def append(self, measurement: str,
...
+        :param raw_point_list: List which contains multiple measurement points.
+        For more information see function self.append().
```
Comment (OParczyk) [inline src/cloudBuffer/buffer.py:73]:
```suggestion

```

File: src/cloudBuffer/buffer.py [hunk 4]
```diff
@@ -79,6 +105,10 @@ def append_many(self, raw_point_list: List[
         self.__sem.release()
 
     def write_points(self):
+        """
+        Function which writes points into the InfluxDB. If the transmission was
+        successful, the points will be deleted from buffer.
```
Comment (OParczyk) [inline src/cloudBuffer/buffer.py:110]:
```suggestion
        Writes buffer into the InfluxDB. If the transmission was
        successful, the points will be deleted from buffer.
```

File: src/cloudBuffer/buffer.py [hunk 5]
```diff
@@ -92,6 +122,10 @@ def write_points(self):
         self.__sem.release()
 
     def __pop_first(self, number_of_elements: int):
+        """
+        Function which deletes the first n elements from the buffer.
```
Comment (OParczyk) [inline src/cloudBuffer/buffer.py:126]:
```suggestion
        Deletes the first n elements from the buffer.
```

File: src/cloudBuffer/influxWrapper.py [hunk 6]
```diff
@@ -30,12 +38,18 @@ def __init__(self, connection_params: dict):
...
-        self.__influxDBClient = InfluxDBClient(url=self.__url, token=self.__token,
+        self.__influxDBClient = InfluxDBClient(url=self.__url,
+                                               token=self.__token,
                                                org=self.__org)
         self.write_api = self.__influxDBClient.write_api(
             write_options=SYNCHRONOUS)
 
     def insert(self, point: Point):
+        """
+        The function inserts a data point into the InfluxDB.
```
Comment (OParczyk) [inline src/cloudBuffer/influxWrapper.py:49]:
```suggestion
        Inserts a data point into the InfluxDB.
```

File: src/cloudBuffer/influxWrapper.py [hunk 7]
```diff
@@ -30,12 +38,18 @@ def __init__(self, connection_params: dict):
...
+        :param point: Point which should be stored in the InfluxDB
```
Comment (OParczyk) [inline src/cloudBuffer/influxWrapper.py:50]:
```suggestion
        :param point: Point to be stored in the InfluxDB
```

File: src/cloudBuffer/influxWrapper.py [hunk 8]
```diff
@@ -30,12 +38,18 @@ def __init__(self, connection_params: dict):
...
+        :return: 0 if write was successful, non 0 if an error occurred
```
Comment (OParczyk) [inline src/cloudBuffer/influxWrapper.py:51]:
```suggestion
        :return: 0 if write was successful, non-zero if an error occurred
```

File: src/cloudBuffer/influxWrapper.py [hunk 9]
```diff
@@ -47,6 +61,11 @@ def insert(self, point: Point):
             return 1
 
     def insert_many(self, points: List[Point]):
+        """
+        The function inserts multiple data points into the InfluxDB.
```
Comment (OParczyk) [inline src/cloudBuffer/influxWrapper.py:65]:
```suggestion
        Inserts multiple data points into the InfluxDB.
```

File: src/cloudBuffer/influxWrapper.py [hunk 10]
```diff
@@ -47,6 +61,11 @@ def insert(self, point: Point):
...
+        :param points: List of points which should be written to the InfluxDB.
```
Comment (OParczyk) [inline src/cloudBuffer/influxWrapper.py:66]:
```suggestion
        :param points: List of points to be written to the InfluxDB.
```

File: src/cloudBuffer/influxWrapper.py [hunk 11]
```diff
@@ -47,6 +61,11 @@ def insert(self, point: Point):
...
+        :return: 0 if write was successful, non 0 if an error occurred
```
Comment (OParczyk) [inline src/cloudBuffer/influxWrapper.py:67]:
```suggestion
        :return: 0 if write was successful, non-zero if an error occurred
```

**Review by OParczyk — CHANGES_REQUESTED** [review]
Reviewed docstrings, tests need to be removed from this pull request

**Comment by C0ntroller** [comment]
Commits with docstring were cherry-picked.

**Pull request closed without merging**